src/Util/FindReflectionOnLine.php

```diff
@@ -24,21 +23,25 @@
      */
     private $sourceLocator;
 
-    public function __construct(SourceLocator $sourceLocator)
+    /**
+     * @var Locator
+     */
+    private $astLocator;
+
+    public function __construct(SourceLocator $sourceLocator, Locator $astLocator)
     {
         $this->sourceLocator = $sourceLocator;
+        $this->astLocator    = $astLocator;
     }
 
     /**
-     * @return self
+     * @deprecated
```

Comment:
Should we keep this method now that we have a global entry point for the library?